<html><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><title>Research Projects</title><style>
/* cspell:disable-file */
/* webkit printing magic: print all background colors */
html {
	-webkit-print-color-adjust: exact;
}
* {
	box-sizing: border-box;
	-webkit-print-color-adjust: exact;
}

html,
body {
	margin: 0;
	padding: 0;
}
@media only screen {
	body {
		margin: 2em auto;
		max-width: 900px;
		color: rgb(55, 53, 47);
	}
}

body {
	line-height: 1.5;
	white-space: pre-wrap;
}

a,
a.visited {
	color: inherit;
	text-decoration: underline;
}

.pdf-relative-link-path {
	font-size: 80%;
	color: #444;
}

h1,
h2,
h3 {
	letter-spacing: -0.01em;
	line-height: 1.2;
	font-weight: 600;
	margin-bottom: 0;
}

.page-title {
	font-size: 2.5rem;
	font-weight: 700;
	margin-top: 0;
	margin-bottom: 0.75em;
}

h1 {
	font-size: 1.875rem;
	margin-top: 1.875rem;
}

h2 {
	font-size: 1.5rem;
	margin-top: 1.5rem;
}

h3 {
	font-size: 1.25rem;
	margin-top: 1.25rem;
}

.source {
	border: 1px solid #ddd;
	border-radius: 3px;
	padding: 1.5em;
	word-break: break-all;
}

.callout {
	border-radius: 3px;
	padding: 1rem;
}

figure {
	margin: 1.25em 0;
	page-break-inside: avoid;
}

figcaption {
	opacity: 0.5;
	font-size: 85%;
	margin-top: 0.5em;
}

mark {
	background-color: transparent;
}

.indented {
	padding-left: 1.5em;
}

hr {
	background: transparent;
	display: block;
	width: 100%;
	height: 1px;
	visibility: visible;
	border: none;
	border-bottom: 1px solid rgba(55, 53, 47, 0.09);
}

img {
	max-width: 100%;
}

@media only print {
	img {
		max-height: 100vh;
		object-fit: contain;
	}
}

@page {
	margin: 1in;
}

.collection-content {
	font-size: 0.875rem;
}

.column-list {
	display: flex;
	justify-content: space-between;
}

.column {
	padding: 0 1em;
}

.column:first-child {
	padding-left: 0;
}

.column:last-child {
	padding-right: 0;
}

.table_of_contents-item {
	display: block;
	font-size: 0.875rem;
	line-height: 1.3;
	padding: 0.125rem;
}

.table_of_contents-indent-1 {
	margin-left: 1.5rem;
}

.table_of_contents-indent-2 {
	margin-left: 3rem;
}

.table_of_contents-indent-3 {
	margin-left: 4.5rem;
}

.table_of_contents-link {
	text-decoration: none;
	opacity: 0.7;
	border-bottom: 1px solid rgba(55, 53, 47, 0.18);
}

table,
th,
td {
	border: 1px solid rgba(55, 53, 47, 0.09);
	border-collapse: collapse;
}

table {
	border-left: none;
	border-right: none;
}

th,
td {
	font-weight: normal;
	padding: 0.25em 0.5em;
	line-height: 1.5;
	min-height: 1.5em;
	text-align: left;
}

th {
	color: rgba(55, 53, 47, 0.6);
}

ol,
ul {
	margin: 0;
	margin-block-start: 0.6em;
	margin-block-end: 0.6em;
}

li > ol:first-child,
li > ul:first-child {
	margin-block-start: 0.6em;
}

ul > li {
	list-style: disc;
}

ul.to-do-list {
	text-indent: -1.7em;
}

ul.to-do-list > li {
	list-style: none;
}

.to-do-children-checked {
	text-decoration: line-through;
	opacity: 0.375;
}

ul.toggle > li {
	list-style: none;
}

ul {
	padding-inline-start: 1.7em;
}

ul > li {
	padding-left: 0.1em;
}

ol {
	padding-inline-start: 1.6em;
}

ol > li {
	padding-left: 0.2em;
}

.mono ol {
	padding-inline-start: 2em;
}

.mono ol > li {
	text-indent: -0.4em;
}

.toggle {
	padding-inline-start: 0em;
	list-style-type: none;
}

/* Indent toggle children */
.toggle > li > details {
	padding-left: 1.7em;
}

.toggle > li > details > summary {
	margin-left: -1.1em;
}

.selected-value {
	display: inline-block;
	padding: 0 0.5em;
	background: rgba(206, 205, 202, 0.5);
	border-radius: 3px;
	margin-right: 0.5em;
	margin-top: 0.3em;
	margin-bottom: 0.3em;
	white-space: nowrap;
}

.collection-title {
	display: inline-block;
	margin-right: 1em;
}

.simple-table {
	margin-top: 1em;
	font-size: 0.875rem;
	empty-cells: show;
}
.simple-table td {
	height: 29px;
	min-width: 120px;
}

.simple-table th {
	height: 29px;
	min-width: 120px;
}

.simple-table-header-color {
	background: rgb(247, 246, 243);
	color: black;
}
.simple-table-header {
	font-weight: 500;
}

time {
	opacity: 0.5;
}

.icon {
	display: inline-block;
	max-width: 1.2em;
	max-height: 1.2em;
	text-decoration: none;
	vertical-align: text-bottom;
	margin-right: 0.5em;
}

img.icon {
	border-radius: 3px;
}

.user-icon {
	width: 1.5em;
	height: 1.5em;
	border-radius: 100%;
	margin-right: 0.5rem;
}

.user-icon-inner {
	font-size: 0.8em;
}

.text-icon {
	border: 1px solid #000;
	text-align: center;
}

.page-cover-image {
	display: block;
	object-fit: cover;
	width: 100%;
	max-height: 30vh;
}

.page-header-icon {
	font-size: 3rem;
	margin-bottom: 1rem;
}

.page-header-icon-with-cover {
	margin-top: -0.72em;
	margin-left: 0.07em;
}

.page-header-icon img {
	border-radius: 3px;
}

.link-to-page {
	margin: 1em 0;
	padding: 0;
	border: none;
	font-weight: 500;
}

p > .user {
	opacity: 0.5;
}

td > .user,
td > time {
	white-space: nowrap;
}

input[type="checkbox"] {
	transform: scale(1.5);
	margin-right: 0.6em;
	vertical-align: middle;
}

p {
	margin-top: 0.5em;
	margin-bottom: 0.5em;
}

.image {
	border: none;
	margin: 1.5em 0;
	padding: 0;
	border-radius: 0;
	text-align: center;
}

.code,
code {
	background: rgba(135, 131, 120, 0.15);
	border-radius: 3px;
	padding: 0.2em 0.4em;
	border-radius: 3px;
	font-size: 85%;
	tab-size: 2;
}

code {
	color: #eb5757;
}

.code {
	padding: 1.5em 1em;
}

.code-wrap {
	white-space: pre-wrap;
	word-break: break-all;
}

.code > code {
	background: none;
	padding: 0;
	font-size: 100%;
	color: inherit;
}

blockquote {
	font-size: 1.25em;
	margin: 1em 0;
	padding-left: 1em;
	border-left: 3px solid rgb(55, 53, 47);
}

.bookmark {
	text-decoration: none;
	max-height: 8em;
	padding: 0;
	display: flex;
	width: 100%;
	align-items: stretch;
}

.bookmark-title {
	font-size: 0.85em;
	overflow: hidden;
	text-overflow: ellipsis;
	height: 1.75em;
	white-space: nowrap;
}

.bookmark-text {
	display: flex;
	flex-direction: column;
}

.bookmark-info {
	flex: 4 1 180px;
	padding: 12px 14px 14px;
	display: flex;
	flex-direction: column;
	justify-content: space-between;
}

.bookmark-image {
	width: 33%;
	flex: 1 1 180px;
	display: block;
	position: relative;
	object-fit: cover;
	border-radius: 1px;
}

.bookmark-description {
	color: rgba(55, 53, 47, 0.6);
	font-size: 0.75em;
	overflow: hidden;
	max-height: 4.5em;
	word-break: break-word;
}

.bookmark-href {
	font-size: 0.75em;
	margin-top: 0.25em;
}

.sans { font-family: ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol"; }
.code { font-family: "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace; }
.serif { font-family: Lyon-Text, Georgia, ui-serif, serif; }
.mono { font-family: iawriter-mono, Nitti, Menlo, Courier, monospace; }
.pdf .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK JP'; }
.pdf:lang(zh-CN) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK SC'; }
.pdf:lang(zh-TW) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK TC'; }
.pdf:lang(ko-KR) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK KR'; }
.pdf .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK JP'; }
.pdf:lang(zh-CN) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK SC'; }
.pdf:lang(zh-TW) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK TC'; }
.pdf:lang(ko-KR) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK KR'; }
.pdf .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK JP'; }
.pdf:lang(zh-CN) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK SC'; }
.pdf:lang(zh-TW) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK TC'; }
.pdf:lang(ko-KR) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK KR'; }
.pdf .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK JP'; }
.pdf:lang(zh-CN) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK SC'; }
.pdf:lang(zh-TW) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK TC'; }
.pdf:lang(ko-KR) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK KR'; }
.highlight-default {
	color: rgba(55, 53, 47, 1);
}
.highlight-gray {
	color: rgba(120, 119, 116, 1);
	fill: rgba(120, 119, 116, 1);
}
.highlight-brown {
	color: rgba(159, 107, 83, 1);
	fill: rgba(159, 107, 83, 1);
}
.highlight-orange {
	color: rgba(217, 115, 13, 1);
	fill: rgba(217, 115, 13, 1);
}
.highlight-yellow {
	color: rgba(203, 145, 47, 1);
	fill: rgba(203, 145, 47, 1);
}
.highlight-teal {
	color: rgba(68, 131, 97, 1);
	fill: rgba(68, 131, 97, 1);
}
.highlight-blue {
	color: rgba(51, 126, 169, 1);
	fill: rgba(51, 126, 169, 1);
}
.highlight-purple {
	color: rgba(144, 101, 176, 1);
	fill: rgba(144, 101, 176, 1);
}
.highlight-pink {
	color: rgba(193, 76, 138, 1);
	fill: rgba(193, 76, 138, 1);
}
.highlight-red {
	color: rgba(212, 76, 71, 1);
	fill: rgba(212, 76, 71, 1);
}
.highlight-gray_background {
	background: rgba(241, 241, 239, 1);
}
.highlight-brown_background {
	background: rgba(244, 238, 238, 1);
}
.highlight-orange_background {
	background: rgba(251, 236, 221, 1);
}
.highlight-yellow_background {
	background: rgba(251, 243, 219, 1);
}
.highlight-teal_background {
	background: rgba(237, 243, 236, 1);
}
.highlight-blue_background {
	background: rgba(231, 243, 248, 1);
}
.highlight-purple_background {
	background: rgba(244, 240, 247, 0.8);
}
.highlight-pink_background {
	background: rgba(249, 238, 243, 0.8);
}
.highlight-red_background {
	background: rgba(253, 235, 236, 1);
}
.block-color-default {
	color: inherit;
	fill: inherit;
}
.block-color-gray {
	color: rgba(120, 119, 116, 1);
	fill: rgba(120, 119, 116, 1);
}
.block-color-brown {
	color: rgba(159, 107, 83, 1);
	fill: rgba(159, 107, 83, 1);
}
.block-color-orange {
	color: rgba(217, 115, 13, 1);
	fill: rgba(217, 115, 13, 1);
}
.block-color-yellow {
	color: rgba(203, 145, 47, 1);
	fill: rgba(203, 145, 47, 1);
}
.block-color-teal {
	color: rgba(68, 131, 97, 1);
	fill: rgba(68, 131, 97, 1);
}
.block-color-blue {
	color: rgba(51, 126, 169, 1);
	fill: rgba(51, 126, 169, 1);
}
.block-color-purple {
	color: rgba(144, 101, 176, 1);
	fill: rgba(144, 101, 176, 1);
}
.block-color-pink {
	color: rgba(193, 76, 138, 1);
	fill: rgba(193, 76, 138, 1);
}
.block-color-red {
	color: rgba(212, 76, 71, 1);
	fill: rgba(212, 76, 71, 1);
}
.block-color-gray_background {
	background: rgba(241, 241, 239, 1);
}
.block-color-brown_background {
	background: rgba(244, 238, 238, 1);
}
.block-color-orange_background {
	background: rgba(251, 236, 221, 1);
}
.block-color-yellow_background {
	background: rgba(251, 243, 219, 1);
}
.block-color-teal_background {
	background: rgba(237, 243, 236, 1);
}
.block-color-blue_background {
	background: rgba(231, 243, 248, 1);
}
.block-color-purple_background {
	background: rgba(244, 240, 247, 0.8);
}
.block-color-pink_background {
	background: rgba(249, 238, 243, 0.8);
}
.block-color-red_background {
	background: rgba(253, 235, 236, 1);
}
.select-value-color-pink { background-color: rgba(245, 224, 233, 1); }
.select-value-color-purple { background-color: rgba(232, 222, 238, 1); }
.select-value-color-green { background-color: rgba(219, 237, 219, 1); }
.select-value-color-gray { background-color: rgba(227, 226, 224, 1); }
.select-value-color-opaquegray { background-color: rgba(255, 255, 255, 0.0375); }
.select-value-color-orange { background-color: rgba(250, 222, 201, 1); }
.select-value-color-brown { background-color: rgba(238, 224, 218, 1); }
.select-value-color-red { background-color: rgba(255, 226, 221, 1); }
.select-value-color-yellow { background-color: rgba(253, 236, 200, 1); }
.select-value-color-blue { background-color: rgba(211, 229, 239, 1); }

.checkbox {
	display: inline-flex;
	vertical-align: text-bottom;
	width: 16;
	height: 16;
	background-size: 16px;
	margin-left: 2px;
	margin-right: 5px;
}

.checkbox-on {
	background-image: url("data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2216%22%20height%3D%2216%22%20viewBox%3D%220%200%2016%2016%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%0A%3Crect%20width%3D%2216%22%20height%3D%2216%22%20fill%3D%22%2358A9D7%22%2F%3E%0A%3Cpath%20d%3D%22M6.71429%2012.2852L14%204.9995L12.7143%203.71436L6.71429%209.71378L3.28571%206.2831L2%207.57092L6.71429%2012.2852Z%22%20fill%3D%22white%22%2F%3E%0A%3C%2Fsvg%3E");
}

.checkbox-off {
	background-image: url("data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2216%22%20height%3D%2216%22%20viewBox%3D%220%200%2016%2016%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%0A%3Crect%20x%3D%220.75%22%20y%3D%220.75%22%20width%3D%2214.5%22%20height%3D%2214.5%22%20fill%3D%22white%22%20stroke%3D%22%2336352F%22%20stroke-width%3D%221.5%22%2F%3E%0A%3C%2Fsvg%3E");
}
	
</style></head><body><article id="029c1c57-54e0-40a7-9078-7e8e8314a9b5" class="page sans"><header><img class="page-cover-image" src="Research%20Projects%20029c1c5754e040a790787e8e8314a9b5/Screen_Shot_2022-08-30_at_11.47.53_AM.png" style="object-position:center 80.89999999999999%"/><div class="page-header-icon page-header-icon-with-cover"><span class="icon">🧪</span></div><h1 class="page-title">Research Projects</h1></header><div class="page-body"><hr id="66205570-6969-4cc0-a692-3ac5b0db7c06"/><div id="22d7d5fd-4720-4688-aaad-efb53cc22c7a" class="column-list"><div id="d664aa32-f597-4e00-a34b-e7ab4030ebc7" style="width:31.25%" class="column"><p id="bd2cf099-439e-4c97-af22-a7b1fce1c19c" class=""><strong>REPRODUCIBILITY OF SAGA ALGORITHMS  </strong></p><p id="8e2e379f-8005-4a76-b9f0-2d6c019926c9" class="">
</p><p id="c6155a14-b7da-4657-9590-7dd1775caf31" class=""><strong>2021 - 2022  </strong></p><p id="677849c2-1349-4b27-b117-5a88b40837a4" class=""><a href="http://github.com/mehdiforoozandeh/SAGAconf">github.com/mehdiforoozandeh/SAGAconf</a></p><p id="6f618e80-be3c-4b6d-ab36-b704e5ec1611" class="">
</p><p id="e0e06183-a248-4cc0-b19f-0c6c7f850c21" class="">P<em>robabilistic Modeling; Machine Learning; Reproducibility; Epigenomics; Genomics</em></p></div><div id="e1401300-bef6-46c7-a75b-6d50aa4c9aad" style="width:68.75%" class="column"><p id="dcd4d1bc-1722-448c-8017-2fae6a1190be" class="">Research on evaluating the reproducibility of predictions from a well-known class of probabilistic graphical models known as SAGA algorithms. These models, that are usually based on Hidden Markov Models or Dynamic Bayesian Networks, are essentially clustering algorithms for sequence data (genomic and epigenomic data). Similar to most unsupervised learning algorithms, evaluation of their prediction performance is challenging. In this research project, I use statistical principles of reproducibility analysis to evaluate these probabilistic machine learning algorithms.</p><p id="a1185908-6c50-45f5-8be8-d0059699e2aa" class="">
</p><figure id="16efd227-d7b5-406c-b12e-1e90ab9cdec7"><div class="source"><a href="https://www.youtube.com/watch?v=ipnSJxpV-Es">https://www.youtube.com/watch?v=ipnSJxpV-Es</a></div></figure></div></div><p id="e52c2f3d-8c90-49cf-85fb-dd8ad287edb3" class="">
</p><hr id="23dc8538-ec14-45e1-9cc1-af9304c5c7da"/><div id="3719a917-fe60-4096-a903-0d4d4550bc0c" class="column-list"><div id="21122a57-4ff9-4d88-82ce-b2e6b80ecbcc" style="width:31.25%" class="column"><p id="39099467-fc9f-4b1f-8fd1-2a3fafa6b7f6" class=""><strong>DRUG RESISTANCE PREDICTION USING MACHINE LEARNING </strong></p><p id="e218fb4e-1044-4e8e-a244-e94d3fed2096" class="">
</p><p id="72f0f714-20bf-442e-b5ee-fd065b2e5b2e" class=""><strong>2020 - 2021 </strong></p><p id="1433c943-8428-48bb-82c8-7315b4f3e3d8" class=""><a href="http://github.com/mehdiforoozandeh/DRML">github.com/mehdiforoozandeh/DRML
</a></p><p id="64d1e4ab-9096-4da9-ab88-29a6b2cf1bd4" class=""><em>Deep Learning; Bayesian Optimization; Drug Resistance</em></p></div><div id="0eefc926-e9a1-4f64-8a8c-70263cee7e4f" style="width:68.75%" class="column"><p id="18731965-1c43-476a-8fda-c9b627d1c588" class="">In a series of related projects, I designed and optimized several machine learning and deep learning frameworks to understand and predict anti-microbial drug resistance from genomic data. In this project, I explored various avenues such as customized feature extraction from high dimensional genomic data, comparing several machine learning algorithms for the same prediction problem, and most importantly, implementing Bayesian hyper-parameter optimization to optimize our models more effectively. Particularly, I applied Bayesian hyper parameter optimization to optimize a massive deep learning model that could not possibly be optimized using conventional grid-searching method.</p><figure id="61afc083-e4e6-4e4a-b5f4-6661be0f9281"><a href="https://github.com/mehdiforoozandeh/DRML" class="bookmark source"><div class="bookmark-info"><div class="bookmark-text"><div class="bookmark-title">GitHub - mehdiforoozandeh/DRML: Drug Resistance Prediction in Bacteria Using Machine Learning</div><div class="bookmark-description">Drug Resistance Prediction in Escherichia coli Bacteria Using Machine Learning Author: Mehdi Foroozandeh Shahraki Abstract Drug resistance is a growing public health threat with catastrophic medical and economic impacts thus, powerful methodologies are required urgently to tackle this crisis.</div></div><div class="bookmark-href"><img src="https://github.com/favicon.ico" class="icon bookmark-icon"/>https://github.com/mehdiforoozandeh/DRML</div></div><img src="https://opengraph.githubassets.com/bdac5c4b6527061cd6fbf723ced2a776b5e4ffb876be8b680eeedcaa481d8503/mehdiforoozandeh/DRML" class="bookmark-image"/></a></figure><figure id="56a33e06-6d87-4f3e-a662-ceb0a7305579"><a href="https://github.com/mehdiforoozandeh/GFF3toCSV" class="bookmark source"><div class="bookmark-info"><div class="bookmark-text"><div class="bookmark-title">GitHub - mehdiforoozandeh/GFF3toCSV: GFF3 to CSV</div><div class="bookmark-description">You can&#x27;t perform that action at this time. You signed in with another tab or window. You signed out in another tab or window. Reload to refresh your session. Reload to refresh your session.</div></div><div class="bookmark-href"><img src="https://github.com/favicon.ico" class="icon bookmark-icon"/>https://github.com/mehdiforoozandeh/GFF3toCSV</div></div><img src="https://opengraph.githubassets.com/53842628119d9bd13e3c901082f38e17390eeb7a81a37cb41d920a339cedceef/mehdiforoozandeh/GFF3toCSV" class="bookmark-image"/></a></figure></div></div><p id="e89c743d-1021-4060-8f00-2206c17eafe4" class="">
</p><hr id="f0445565-1393-4361-8250-b294d1e3293a"/><div id="98e6b07d-5bcd-4df0-bcf6-9de96a1ca978" class="column-list"><div id="183bca80-a35a-4809-ada4-10250e22c6df" style="width:31.25%" class="column"><p id="abb51fb8-526e-4649-b6f2-006d28096e6e" class=""><strong>PREIDCTION OF ENZYMATIC PROPERTIES USING MACHINE LEARNING AND ENZYME MINING FROM METAGENOME  </strong></p><p id="9cd7184e-3c15-4f70-9863-f012e7368b15" class="">
</p><p id="ad33f1c7-4aa4-4670-811e-724d7ac8e25b" class=""><strong>2019-2020 </strong></p><p id="de8f0d4f-2d2b-4487-a573-b196a54f0e97" class=""><a href="http://github.com/mehdiforoozandeh/MeTarEnz">github.com/mehdiforoozandeh/MeTarEnz </a></p><p id="3c28ca9c-b8b4-442c-8fea-4fd413298df2" class=""><a href="http://github.com/mehdiforoozandeh/MCIC">github.com/mehdiforoozandeh/MCIC</a></p><p id="d53fcf38-edbe-493c-81e6-581cd0008b9e" class=""><a href="https://github.com/mehdiforoozandeh/TAXyl">https://github.com/mehdiforoozandeh/TAXyl</a></p><p id="ffaf7329-817a-4f73-ad4e-2cc68f3b48c0" class="">
</p><p id="b5b2add7-892c-4b91-890a-1dd0091954d7" class=""><em>Enzyme mining; Metagenomics; Machine Learning; Automation</em></p></div><div id="e47fd2f3-8cac-412e-96b8-91e04aeeb5e6" style="width:68.75%" class="column"><p id="bbb92d2a-8d57-421c-9808-c4ef00c5bb47" class="">Research on developing machine learning and deep learning models for predicting enzyme properties heavily relying on amino acid sequence data for model training and feature extraction. I also designed and developed automated enzyme mining software to facilitate discovery of biocatalysts from metagenome data. Finally, using the developed computational infrastructures, we discovered and mined various novel enzymes with particular applications from metagenomic sources.</p><p id="533500ac-891e-40b3-9971-31c3b3503d3d" class="">
</p><figure id="3e7718ce-0257-4986-b4ba-da267e50887e"><a href="https://github.com/mehdiforoozandeh/MeTarEnz" class="bookmark source"><div class="bookmark-info"><div class="bookmark-text"><div class="bookmark-title">GitHub - mehdiforoozandeh/MeTarEnz: Metagenomic Targeted Enzyme Miner</div><div class="bookmark-description">MeTarEnz (metagenomic targeted enzyme miner) is a multi-service software that enables the targeted screening of high-throughput metagenomic data with user-defined databases and bit-score cut-offs. Moreover, MeTarEnz contains an assembly module making it capable of providing its screening services while accepting various types of input data such as raw reads, assembled contigs, and translated protein sequences.</div></div><div class="bookmark-href"><img src="https://github.com/favicon.ico" class="icon bookmark-icon"/>https://github.com/mehdiforoozandeh/MeTarEnz</div></div><img src="https://opengraph.githubassets.com/c85ea320cc6cafdfa78ac195dc3f388cc224307f8c5bcfc6cffcc53bd3ab8fe6/mehdiforoozandeh/MeTarEnz" class="bookmark-image"/></a></figure><figure id="6114a6fa-174e-4972-bc93-d8474933177a"><a href="https://github.com/mehdiforoozandeh/MCIC" class="bookmark source"><div class="bookmark-info"><div class="bookmark-text"><div class="bookmark-title">GitHub - mehdiforoozandeh/MCIC: MCIC is a software developed with the primary purpose of identifying cellulose-degrading enzymes in large metagenomic data followed by the characterization of their pH and temperature dependence using trained machine learning models.</div><div class="bookmark-description">MCIC is a software developed with the primary purpose of identifying cellulose-degrading enzymes in large metagenomic data followed by the characterization of their pH and temperature dependence using trained machine learning models. - GitHub - mehdiforoozandeh/MCIC: MCIC is a software developed with the primary purpose of identifying cellulose-degrading enzymes in large metagenomic data followed by the characterization of their pH and temperature dependence using trained machine learning models.</div></div><div class="bookmark-href"><img src="https://github.com/favicon.ico" class="icon bookmark-icon"/>https://github.com/mehdiforoozandeh/MCIC</div></div><img src="https://opengraph.githubassets.com/b44be6ce3418ad9df65b8899d1c83c7c62cc42f4aa979dacf96fa861f3ae25cd/mehdiforoozandeh/MCIC" class="bookmark-image"/></a></figure><figure id="9e71aa86-4c00-4819-ad92-2bf46da34294"><a href="https://github.com/mehdiforoozandeh/TAXyl" class="bookmark source"><div class="bookmark-info"><div class="bookmark-text"><div class="bookmark-title">GitHub - mehdiforoozandeh/TAXyl: TAXyl (Thermal activity prediction for xylanase)</div><div class="bookmark-description">TAXyl (Thermal activity prediction for xylanase) Growing industrial utilization of enzymes, and the increasing availability of metagenomic data highlights the demand for effective methods of targeted identification and verification of novel enzymes from various environmental microbiota. Xylanases are a class of enzymes with numerous industrial applications and are involved in the degradation of xylose, a component of lignocellulose.</div></div><div class="bookmark-href"><img src="https://github.com/favicon.ico" class="icon bookmark-icon"/>https://github.com/mehdiforoozandeh/TAXyl</div></div><img src="https://opengraph.githubassets.com/a4751c93aaa5618817e38f25369cc29ea27a11de507e786dcf5e4a943a12a25e/mehdiforoozandeh/TAXyl" class="bookmark-image"/></a></figure></div></div><hr id="dc0a637d-856f-4352-ba4d-ad01946110ce"/><div id="6818b342-863e-453a-9e7d-ab7d20b9ca18" class="column-list"><div id="2764add0-b47e-43ff-a45a-6b6bac2215e5" style="width:31.25%" class="column"><p id="26b87c3a-8fac-4771-88db-0f74f845e262" class=""><strong>Mining Novel Industrial Enzymes from Metagnomic Sources</strong></p><p id="3c18e9e5-b394-4cd9-af39-d5f626fe3ae2" class="">
</p><p id="8167bfde-e518-4f46-8f13-32ae88fc6122" class=""><strong>2019-2020 </strong></p><p id="ff99e7e6-da04-4972-a292-2331e62190e9" class=""><a href="http://github.com/mehdiforoozandeh/MeTarEnz">github.com/mehdiforoozandeh/MeTarEnz </a></p><p id="7cf30f7b-9632-4c3b-b50f-194c61549785" class=""><a href="http://github.com/mehdiforoozandeh/MCIC">github.com/mehdiforoozandeh/MCIC</a></p><p id="56842ea2-b012-41e6-9489-11f1fdb35997" class=""><a href="https://github.com/mehdiforoozandeh/TAXyl">https://github.com/mehdiforoozandeh/TAXyl</a></p><p id="d79d0908-7f92-4b7f-b207-d930a3c3a821" class="">
</p><p id="a3e9c62a-3e05-4b57-835c-b9f4ea78dd49" class=""><em>Enzyme mining; Metagenomics;</em></p></div><div id="f02b43ce-dbfb-4624-b091-ae789a986520" style="width:68.75%" class="column"><p id="15577a68-4c98-4393-b289-498d0f136365" class="">Research on the discovery of novel enzymes with particular applications from metagenomic sources</p><p id="076284b9-5ec3-40cb-8054-6e8269fb1b31" class="">
</p><p id="e915b41b-da3a-4e4b-a7dc-e3720275ad52" class="">Papers:</p><ul id="a896ceec-c361-4a37-83a0-8a285059adab" class="bulleted-list"><li style="list-style-type:disc">Maleki, Morteza, <em>et al. </em>”A novel thermostable cellulase cocktail enhances lignocellulosic bioconversion and biorefining in a broad range of pH.” International journal of biological macromolecules 154 (2020) : 349-360.</li></ul><ul id="8ddeba3f-15a7-40c2-b042-bdc20fb180b4" class="bulleted-list"><li style="list-style-type:disc">Ariaeenejad, Shohreh, <em>et al. </em>
”A novel high performance in-silico screened metagenome-derived alkali-thermostable endo-β-1, 4-glucanase for lignocellulosic biomass hydrolysis in the harsh conditions.” BMC biotechnology 20.1 (2020) :</li></ul><p id="e5ef4330-d179-4f79-96f0-4533155774b0" class="">
</p></div></div><hr id="073e36ce-cbb7-42bf-8ebe-845c6e26750d"/><p id="a21e0833-b06d-4473-8f9a-36e2f1f58d6d" class="">
</p></div></article></body></html>