<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<meta name="viewport"    content="width=device-width, initial-scale=1.0">
	<meta name="description" content="Personal Webpage of Mehdi Foroozandeh">
	<meta name="author"      content="Mehdi Foroozandeh">
	
	<title>Mehdi Foroozandeh</title>

	<link rel="shortcut icon" href="assets/images/gt_favicon.png">
	
	<!-- Bootstrap -->
	<link href="https://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.no-icons.min.css" rel="stylesheet">
	<!-- Icons -->
	<link href="https://netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css" rel="stylesheet">
	<!-- Fonts -->
	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Alice|Open+Sans:400,300,700">
	<!-- Custom styles -->
	<link rel="stylesheet" href="assets/css/styles.css">

	<!--[if lt IE 9]> <script src="assets/js/html5shiv.js"></script> <![endif]-->
</head>
<body class="home">

<header id="header">
	<div id="head" class="parallax" parallax-speed="2">
		<h1 id="logo" class="text-center">
			<img class="img-circle" src="assets/images/mforooz_avatar.jpg" alt="">
			<span class="title">Mehdi Foroozandeh</span>
			<span class="tagline">Computational Biology and Machine Learning Enthusiast | Computing Science Ph.D. Student <br>
				<!-- <a href=""> mfa76 <at> sfu <dot> ca  </a></span> -->
		</h1>
	</div>

	<nav class="navbar navbar-default navbar-sticky">
		<div class="container-fluid">
			
			<div class="navbar-er">
				<button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
			</div>
			
			<div class="navbar-collapse collapse">
				
				<ul class="nav navbar-nav">
					<li class="active"><a href="index.html">Home</a></li>
					<li><a href="about.html">About</a></li>
					<li><a href="research.html">Research</a></li>
					<!-- <li class="dropdown">
						<a href="#" class="dropdown-toggle" data-toggle="dropdown">More Pages <b class="caret"></b></a>
						<ul class="dropdown-menu">
							<li><a href="sidebar-left.html">Left Sidebar</a></li>
							<li><a href="sidebar-right.html">Right Sidebar</a></li>
							<li><a href="single.html">Blog Post</a></li>
						</ul>
					</li> -->

					<li><a href="blog.html">Blog</a></li>
				</ul>
			
			</div><!--/.nav-collapse -->			
		</div>	
	</nav>
</header>

<main id="main">

	<div class="container">
		
		<div class="row section topspace">
			<div class="col-md-12">
				<p class="lead text-center text-muted"><a href="assets/CV.pdf" download>Download my CV</a></p>
				<!-- </p><hr id="542eb538-9f49-41e3-a893-e47c161b7fa6"/><h1 id="ae244918-bac6-4e31-821c-9abbbf985ac8" class="">üìö Education</h1><div id="a87f0cbf-939f-489a-8937-0a28b97e77ce" class="column-list"><div id="200136dc-76e3-4ae0-b988-c4832558f88c" style="width:18.75%" class="column"><p id="746d666d-be9f-4641-885b-295b9beb5f06" class="">2022 - 2026</p></div><div id="f03679a0-247e-4cda-9043-6fb18d777719" style="width:81.25%" class="column"><p id="7b4ced64-db2a-4a66-b8ce-fbab557c3c53" class="">Ph.D. Computing Science, Simon Fraser University</p></div></div><div id="1826a0e9-2ac5-4e29-83bc-e89d956f33c6" class="column-list"><div id="23cdfe5f-112a-4f51-adb5-b381e367c47e" style="width:18.75%" class="column"><p id="d252ce04-c222-4595-a8ab-529810fe8d55" class="">2020 - 2022</p></div><div id="17357eb9-e2ca-434e-b43a-1d79d05b8e79" style="width:81.25%" class="column"><p id="7e1a726e-8239-4da3-bc87-308b34f4901c" class="">MSc. Computing Science, Simon Fraser University, </p><p id="118fcd79-37ec-4eb2-aec8-75cccc2b9722" class=""><em>CGPA : 4.0 /4.33</em></p></div></div><div id="8da21060-cb1b-42c8-ba9a-65cf2c9816ba" class="column-list"><div id="0a49adf8-9b1f-4b4d-ace3-c82753562fd7" style="width:18.75%" class="column"><p id="11a8d97d-e341-46db-8109-58124cb6f27c" class="">2015 - 2019</p></div><div id="b693db4c-b7ba-4db4-bf2e-7dcc2904ff15" style="width:81.25%" class="column"><p id="94dc675e-dbd1-4f90-a4a0-0a0bc7010ce5" class="">BSc. Cell and Molecular Biology- Microbiology, University of Tehran, <em>GPA (Final Two Years) : 3.68 /4</em></p></div></div><p id="0ae9251a-76a6-495f-8a04-a3bd4c734ad9" class="">
				</p><hr id="39bb2788-ad04-481c-8d1c-bc86608acd3b"/><h1 id="cb3055be-e1c5-4c7f-b88e-9e3da81e34fd" class="">üéí Experience </h1><div id="ed2ebc52-857a-4673-8cf5-af47b9f6de04" class="column-list"><div id="6f5677bf-84bc-4283-b8ea-e27d68e4d47e" style="width:25%" class="column"><p id="9857df93-5072-4b76-b3fd-47443d47d2d9" class="">September 2020 - Present</p></div><div id="e6e7de32-5993-49b4-a68b-7063559b7779" style="width:75%" class="column"><p id="7fdb81cc-23cb-4661-b33a-0fc813f69bae" class="">Research Assistant, SIMON FRASER UNIVERISTY, Department of Computing Sciences</p><p id="ab549497-820e-4029-8c4d-8a631542e493" class="">
				</p><ul id="a50b24ad-12b2-449e-9f0d-3ddf63df49e4" class="bulleted-list"><li style="list-style-type:disc">At Libbrecht Computational Biology Lab</li></ul><ul id="c4a373aa-9d5e-4bbd-9dcf-ddfd5d74af85" class="bulleted-list"><li style="list-style-type:disc">Research on evaluating the reproducibility of predictions of a class of probabilistic graphical models (SAGA methods).</li></ul><ul id="f7887470-f0aa-47ce-a753-e01aed5566c1" class="bulleted-list"><li style="list-style-type:disc">Research on building and optimizing deep learning models capable of predicting bacterial antibiotic resistance from genome sequence data.</li></ul><ul id="e2ed4079-c91a-447c-82f7-57215d42ab28" class="bulleted-list"><li style="list-style-type:disc">Machine Learning Probabilistic Modeling Genomics Epigenomics Deep Learning Drug Resistance</li></ul></div></div><p id="ada94376-5487-4e00-9a2b-9827fe3e055b" class="">
				</p><div id="2b6efa35-60a5-4d84-a220-2483dc189ff9" class="column-list"><div id="305a4ea3-ab65-4513-b2b7-8579f2c4975a" style="width:25%" class="column"><p id="80e3acec-1094-4fd3-86a0-e42a09afb138" class="">January 2019 - September 2020</p></div><div id="88360272-da2a-479d-a196-0a135d0ab620" style="width:75%" class="column"><p id="655ad2c4-ef7b-432c-a9ae-fb84862afbd6" class="">Research Assistant, UNIVERSITY OF TEHRAN, IBB- Department of Bioinformatics</p><p id="992a1afe-342b-4c96-972f-59689ffebbbe" class="">
				</p><ul id="0dc24d64-316c-42d8-bdfb-4b0ea1cf5d86" class="bulleted-list"><li style="list-style-type:disc">At Complex Biological Systems and Bioinformatics (CBB) lab, led by Dr.Kavousi.</li></ul><ul id="c7285cbd-a5c3-46ee-8b76-98b7b1925e26" class="bulleted-list"><li style="list-style-type:disc">Research on developing machine learning and deep learning models for predicting enzyme properties.</li></ul><ul id="01af041a-935a-4882-bb69-5f91f8528f31" class="bulleted-list"><li style="list-style-type:disc">Research on developing machine learning and deep learning models for predicting enzyme properties.</li></ul><ul id="2c7e5c4a-95a3-4682-9fac-aa4d2c17bc2c" class="bulleted-list"><li style="list-style-type:disc">Research on the discovery of novel enzymes with particular applications from metagenomic sources</li></ul><ul id="98737867-3947-40ad-8ef1-048eebaaf581" class="bulleted-list"><li style="list-style-type:disc">Collaboration in writing and editing academic publications</li></ul><ul id="844efffd-1e2a-4eed-b922-8e21964d8ea8" class="bulleted-list"><li style="list-style-type:disc">Machine Learning Metagenomics Enzymes Bioinformatics</li></ul></div></div><hr id="e7a75841-8e97-4c63-8279-ac054663b76d"/><h1 id="3f7d0c84-ac2c-46ab-90cb-59152f48eb70" class=""><strong>üë©üèª‚Äçüíª</strong> Projects</h1><p id="5e392f8c-3776-4073-8f25-82066725ca6c" class=""><strong>REPRODUCIBILITY OF SAGA ALGORITHMS </strong></p><p id="df2fda7f-450d-4156-8712-5986d67a0ead" class=""><strong>2021 - 2022 </strong></p><p id="1e5d04a1-9421-4992-a528-f6f929f64e0e" class="">github.com/mehdiforoozandeh/SAGAconf
				Research on evaluating the reproducibility of predictions from a well-known class of probabilistic graphical models known as SAGA algorithms. These models, that are usually based on Hidden Markov Models or Dynamic Bayesian Networks, are essentially clustering algorithms for sequence data (genomic and epigenomic data). Similar to most unsupervised learning algorithms, evaluation of their prediction performance is challenging. In this research project, I use statistical principles of reproducibility analysis to evaluate these probabilistic machine learning algorithms.</p><p id="577a946e-5d5f-449d-87cc-3c0e3a38e784" class="">probabilistic modeling machine learning reproducibility epigenomics genomics</p><p id="bf0786cf-2fca-458a-9c4c-d359b8d61a78" class="">
				</p><p id="9883c280-ea6f-40d9-9552-348805700a65" class="">
				</p><p id="45416054-5148-48f8-a459-6b981297ae94" class=""><strong>DRUG RESISTANCE PREDICTION USING MACHINE LEARNING</strong></p><p id="57b325c4-8825-42ac-af3c-faba6dfd579e" class=""><strong>2020 - 2021</strong></p><p id="6a72b429-dd83-4c88-849d-f269e5deb6e7" class="">github.com/mehdiforoozandeh/DRML
				In a series of related projects, I designed and optimized several machine learning and deep learning frameworks to understand and predict anti-microbial drug resistance from genomic data. In this project, I explored various avenues such as customized feature extraction from high dimensional genomic data, comparing several machine learning algorithms for the same prediction problem, and most importantly, implementing Bayesian hyper-parameter optimization to optimize our models more effectively. Particularly, I applied Bayesian hyper parameter optimization to optimize a massive deep learning model that could not possibly be optimized using conventional grid-searching method.</p><p id="b3751c09-2790-4c0a-9c78-a7d6e9675e93" class="">Deep Learning Bayesian Optimization Drug Resistance</p><p id="bd0a9d76-086f-4a96-a786-ca1a7d7f828d" class="">
				</p><p id="3b3f896c-92f9-40d5-b287-51c95652b956" class="">
				</p><p id="5ba541bf-8df5-4c6d-a674-f81d6c80437f" class=""><strong>PREIDCTION OF ENZYMATIC PROPERTIES USING MACHINE LEARNING AND ENZYME MINING FROM METAGENOME </strong></p><p id="9f88af66-3390-48db-936f-7056832d2468" class=""><strong>2019-2020
				</strong>github.com/mehdiforoozandeh/MeTarEnz</p><p id="a4b8f14d-e712-4429-a0d8-678f2b05c89b" class="">github.com/mehdiforoozandeh/MCIC</p><p id="0cd671b4-777f-4f3c-8e8a-5cd49f4ac8a5" class="">
				Research on developing machine learning and deep learning models for predicting enzyme properties heavily relying on amino acid sequence data for model training and feature extraction. I also designed and developed automated enzyme mining software to facilitate discovery of biocatalysts from metagenome data. Finally, using the developed computational infrastructures, we discovered and mined various novel enzymes with particular applications from metagenomic sources.</p><p id="cffa66ea-169c-4976-9515-6b0cadd766ee" class="">Enzyme mining Metagenomics Machine Learning Automation</p><hr id="0f5b89d2-4318-483a-bcc8-771b07f08445"/><h1 id="f5eeaaa2-c279-4537-8d1d-099472702f0e" class="">üõ† Skills</h1><p id="87741bb9-636c-484d-badd-6f3a96841f37" class="">
				</p><div id="36e3f638-6a6f-4a61-bcea-e0fb8d961d5c" class="column-list"><div id="396340c2-ddba-4792-8729-fccbfb14d1aa" style="width:25%" class="column"><p id="38c5ea6c-f6a4-41eb-804c-c5c0b6b90ba4" class="">Programming</p></div><div id="a0053c1d-a787-4912-bcda-eed4c12b12a2" style="width:75%" class="column"><p id="44ff50f2-cf59-492e-a421-02df8c181ae5" class="">Python, MATLAB</p></div></div><div id="e4ad958a-7eeb-437f-9c61-9cfca029676c" class="column-list"><div id="33379fe3-a3d5-40ff-baf6-a3d1443adebd" style="width:25%" class="column"><p id="139980d7-c803-41cf-a070-4fe097fb1ffa" class="">Tools and Software</p></div><div id="ab9a000e-619e-42ce-800d-e5bb443dd354" style="width:75%" class="column"><p id="72ed55b7-2e56-4aa1-9cc6-7474f04b62bc" class="">TensorFlow, Keras, Scikit-learn, Scikit-optimize, Numpy, Pandas, Biopython, NCBI Toolkit, Segway, ChromHMM</p></div></div><div id="24a96f16-f58a-4ff1-a58f-7b391b1cee27" class="column-list"><div id="e828a75c-803c-4c06-81b8-3d54a0fe8070" style="width:25%" class="column"><p id="ae0c3163-1b0c-4827-9493-1430335cd36a" class="">Other</p></div><div id="e0e16831-4725-476e-b2c0-e1ebe6c6fe49" style="width:75%" class="column"><p id="13c5b312-a900-4ef7-9a5d-a5fc903bc67b" class="">Unix Shell Scripting, Docker, LATEX, Linux, Web Scraping
				</p></div></div><div id="cf0bc6a8-ba00-4334-bc1f-89f291d5554a" class="column-list"><div id="b9c5afe4-7e95-4509-a84c-7bb15132510a" style="width:25%" class="column"><p id="4130a658-c565-4580-b557-c4e1bfdfecef" class="">
				Laboratory Skills</p></div><div id="3505382a-f158-4e54-8c5b-53eee4ce985d" style="width:75%" class="column"><p id="e9fb6a86-8062-44e6-a57b-7bdfb750814f" class="">Microbial Culture, PCR, DNA extraction, Gene Cloning, Gel Electrophoresis
				</p></div></div><div id="0730639e-2d3b-4b56-b4db-e165009bf29f" class="column-list"><div id="b2ed7aa1-8020-4823-8bd3-35072ef21b0d" style="width:25%" class="column"><p id="ac2b37bf-e33d-484c-9144-40cb30734a20" class="">Languages</p></div><div id="16bc395a-ac0b-4f72-bd03-a5fe1c4ed3cf" style="width:75%" class="column"><p id="285a33e2-645c-4ee2-9f42-b9bb4ba47434" class="">English, Persian (Native)</p></div></div><hr id="7952d2ab-2a42-408e-975c-3d87cb063eee"/><h1 id="ec9b2eec-99a9-4cd1-b5a2-727df917442c" class="">üìú Publications</h1><ul id="74a6453e-1c1f-4cff-bc8f-5edcc88dcbb2" class="bulleted-list"><li style="list-style-type:disc">Foroozandeh Shahraki, Mehdi, <em>et al. </em>‚ÄùA computational learning paradigm to targeted discovery of biocatalysts from metagenomic data : a case study of lipase identification‚Äù Biotechnology and Bioengineering :21-130.R1 (2022).</li></ul><p id="de51c099-e454-4ac9-b932-49fe5d0a2bae" class="">
				</p><ul id="44b613c5-af0e-4ed8-815e-9f07f07c1304" class="bulleted-list"><li style="list-style-type:disc">Foroozandeh Shahraki, Mehdi, <em>et al. </em>‚ÄùA generalized machine-learning aided method for targeted identification of industrial enzymes from metagenome : A xylanase temperature dependence case study.‚Äù Biotechnology and Bioengineering (2020).</li></ul><p id="39cc61c0-247e-485d-9457-dd386ec0ac7e" class="">
				</p><ul id="ddbe8097-d8b1-4b36-b069-35481bd63194" class="bulleted-list"><li style="list-style-type:disc">Foroozandeh Shahraki, Mehdi, <em>et al. </em>‚ÄùMCIC : Automated Identification of Cellulases From Metagenomic Data and Characterization Based on Temperature and pH Dependence.‚Äù Frontiers in microbiology 11 (2020).</li></ul><p id="d7d812ce-f093-4065-bf2b-19cc8523f990" class="">
				</p><ul id="61271599-1184-47ac-a86f-d866ef477e47" class="bulleted-list"><li style="list-style-type:disc">Maleki, Morteza, <em>et al. </em>‚ÄùA novel thermostable cellulase cocktail enhances lignocellulosic bioconversion and biorefining in a broad range of pH.‚Äù International journal of biological macromolecules 154 (2020) : 349-360.</li></ul><p id="e3d47019-aa00-4dec-a978-f221797c692a" class="">
				</p><ul id="5b8e60aa-1423-4864-a990-d740c5e2c44f" class="bulleted-list"><li style="list-style-type:disc">Ariaeenejad, Shohreh, <em>et al. </em>‚ÄùA novel high performance in-silico screened metagenome-derived alkali-thermostable endo-Œ≤-1, 4-glucanase for lignocellulosic biomass hydrolysis in the harsh conditions.‚Äù BMC biotechnology 20.1 (2020) : 1-13.</li></ul><p id="868acc8a-24cb-419a-8ecf-9c81e3a5c552" class="">
				</p><ul id="c0441a46-92ca-4fc4-bbe8-86d73bad150b" class="bulleted-list"><li style="list-style-type:disc">Foroozandeh Shahraki, Mehdi, <em>et al. </em>‚ÄùEvaluating the reproducibility of segmentation and genome annotation (SAGA) algorithms‚Äù Poster at Machine Learning in Computational Biology 2021 (full submission in progress).</li></ul><hr id="d7ede6eb-3b4c-469e-9905-6ce7a5d3e121"/><p id="689d9193-4e93-46bd-91ab-cb3002efef97" class=""> -->

			</div>

</main>

<footer id="footer">
	<div class="container">
		<div class="row">
			<div class="col-md-3 widget">
				<h3 class="widget-title">Contact</h3>
				<div class="widget-body">
					<p>+1 778 228 2027<br>
						<a href="mailto:#">mfa76@sfu.ca</a><br>
						<a href="mailto:#">mehdiforoozandehsh@gmail.com</a><br>
						<br>
						ASB 9000 - 8888 University Drive. Burnaby, British Columbia V5A 1S6, Canada
					</p>	
				</div>
			</div>

			<div class="col-md-3 widget">
				<h3 class="widget-title"></h3>
				<div class="widget-body">
					<!-- <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque, nihil natus explicabo ipsum quia iste aliquid repellat eveniet velit ipsa sunt libero sed aperiam id soluta officia asperiores adipisci maxime!</p> -->
				</div>
			</div>
			<div class="col-md-3 widget">
				<h3 class="widget-title"></h3>
				<div class="widget-body">
					<!-- <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque, nihil natus explicabo ipsum quia iste aliquid repellat eveniet velit ipsa sunt libero sed aperiam id soluta officia asperiores adipisci maxime!</p> -->
				</div>
			</div>

			<div class="col-md-3 widget">
				<h3 class="widget-title">Follow me</h3>
				<div class="widget-body">
					<p class="follow-me-icons">
						<a href="https://twitter.com/mforoozz"><i class="fa fa-twitter fa-2"></i></a>
						<!-- <a href=""><i class="fa fa-dribbble fa-2"></i></a> -->
						<a href="https://github.com/mehdiforoozandeh"><i class="fa fa-github fa-2"></i></a>
						<a href="https://www.facebook.com/mehdi.foroozandeh.3"><i class="fa fa-facebook fa-2"></i></a>
					</p>
				</div>
			</div>

		</div> 
	</div>
</footer>

<footer id="underfooter">
	<div class="container">
		<div class="row">
			
			<div class="col-md-6 widget">
				<div class="widget-body">
					<p>ASB 9000 - 8888 University Drive. Burnaby, British Columbia V5A 1S6, Canada </p>
				</div>
			</div>

			<div class="col-md-6 widget">
				<div class="widget-body">
					<p class="text-right">
						Copyright &copy; 2022, Mehdi Foroozandeh<br> 
						Design: <a href="http://www.gettemplate.com" rel="designer">Initio by GetTemplate</a> </p>
				</div>
			</div>

		</div> <!-- /row of widgets -->
	</div>
</footer>



<!-- JavaScript libs are placed at the end of the document so the pages load faster -->
<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
<script src="http://netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js"></script>
<script src="assets/js/template.js"></script>
</body>
</html>
